understanding/20/use-of-color.html

```diff
@@ -81,7 +81,29 @@ <h2>Intent of Use of Color</h2>
          </p>
 
       </div>
-      
+
+      <div class="note">
+
+         <p>Under certain conditions, most user agents provide users with a color-only cue that a link has been activated by them ("visited"). However, several technical constraints result in authors having very limited control over this these color-only indications of visited links.  The technical constraints are as follows:
+            <ol>
+               <li> 
+                  User agents constrain the exposure of a link's visited state due to <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/Privacy_and_the_:visited_selector">privacy concerns</a>. Author queries to user agents will indicate all links have not been visited.
+               </li>
+               <li>
+                  Any available information on the visited state of a link would be inaccurate since it is both user and browser-dependent. Even if an author could accurately get information on previously activated links by a certain user, the author would be constrained to the current browser's preserved history, and would be unable to determine if the user had visited the page using a different browser (or if the history was not preserved, either from cache clearing or use of private sessions).
+               </li>
+               <li>
+                  Authors can <em>only</em> use color to modify the <code>:visited</code> CSS psuedoclass style. The technology constrains any non-color styling. Due to palette limitations, an author cannot use color alone to achieve 3:1 contrast between link and non-line text as well as between visited and unvisited links while also achieving 4.5:1 contrast for all link and non-link text.
+               </li>
+               <li>
+                  Authors also cannot set the visited state of links. The anchor element does not include a "visited" attribute; therefore the author has no ability to alter the state through an attribute setting. As such, authors cannot achieve <a href="info-and-relationships">1.3.1 Info and Relationships</a> or
+                  <a href="name-role-value">4.1.2 Name, Role, Value</a> in regard to visited links.
+               </li>            
+            </ol>
+         </p>
+
+         <p>For these reasons, setting or conveying a link's visited status is not an author responsibility, and does not result in a failure of Use of Color</p>
```

Comment:
maybe change `failure of Use of Color` to `failure of this Success Criterion` (similar to how many other SC understandings self-reference)